src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -110,6 +110,26 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 			StringAssertEx.DoesNotContainRegex (@$"Using profile data file.*{filename}\.aotprofile", b.LastBuildOutput, "Should not use default AOT profile", RegexOptions.IgnoreCase);
 		}
 
+		[Test]
+		[Category ("SmokeTests")]
```

Comment:
Going forward, we probably do not need to add any tests to `SmokeTests`, this only affects the tests run on the in-tree Windows build which only runs the bare minimum.

All tests are run against the Mac/Win installers going forward, there are no longer separate Smoke Test jobs.

https://github.com/xamarin/xamarin-android/pull/7872